### ISSUE TITLE
a UIKnob derivative can't have @ScriptProperty things, which is annoying

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#636]
If I try making a UICondKnob (custom class) derive from UIKnob and also have @ScriptProperty, then it doesn't build.

```d
Error: template instance `AliasSeq!(trailDisabled, knobRadius, knobDiffuse, knobMaterial, style, numLEDs, LEDRadiusMin, LEDRadiusMax, LEDDiffuseLit, LEDDiffuseUnlit, LEDMaterial, LEDDistanceFromCenter, LEDDistanceFromCenterDragged, LEDDepth, hasTrail, litTrailDiffuse, unlitTrailDiffuse, trailRadiusMin, trailRadiusMax, trailOffsetX, trailOffsetY, trailMinAngle, trailBaseAngle, trailMaxAngle, litTrailDiffuseAlt, hasAlternateTrail, animationTimeConstant)` `AliasSeq!(trailDisabled, knobRadius, knobDiffuse, knobMaterial, style, numLEDs, LEDRadiusMin, LEDRadiusMax, LEDDiffuseLit, LEDDiffuseUnlit, LEDMaterial, LEDDistanceFromCenter, LEDDistanceFromCenterDragged, LEDDepth, hasTrail, litTrailDiffuse, unlitTrailDiffuse, trailRadiusMin, trailRadiusMax, trailOffsetX, trailOffsetY, trailMinAngle, trailBaseAngle, trailMaxAngle, litTrailDiffuseAlt, hasAlternateTrail, animationTimeConstant)` is nested in both `UICondKnob` and `UIKnob`
```

**Comment by p0nce** [comment]
Reported in D Bugzilla in https://issues.dlang.org/show_bug.cgi?id=22992

Adam suggested a workaround that also makes less template instantiations: https://forum.dlang.org/post/zjysmricjnoytqndgunr@forum.dlang.org

**Comment by p0nce** [comment]
Fixed in v12.4.7
